### ISSUE TITLE
Center chart title

### ISSUE THREAD

**Issue by bwpedro** [rakannimer/react-google-charts#371]
Hello, I'm using Pie Chart for my project and couldn't find a way to center the title of the chart, it is always aligned to the left. I was wondering if there a prop or anything that could center it?

Examples always aligned to the left:
https://react-google-charts.com/pie-chart

**Comment by brunomaffei** [comment]
any solution for this?

**Comment by bwpedro** [comment]
> any solution for this?

not that I know of

**Comment by brunomaffei** [comment]
yeah I was looking for something ...but not working at all, I hope they can fix it or make it a feature haha

**Comment by takkaran** [comment]
> Hello, I'm using Pie Chart for my project and couldn't find a way to center the title of the chart, it is always aligned to the left. I was wondering if there a prop or anything that could center it?
> 
> Examples always aligned to the left: https://react-google-charts.com/pie-chart

Hi i am finding the same. Did u find any solution for it? Please share.
